index.html

```diff
@@ -988,6 +988,10 @@
         </p>
         <p>Special thanks to those that helped pioneer the first edition of this model, including especially: Charu Chandiram, Peter Freiling, Nathan Furtwangler, Thomas Olsen, Matt Rakow, Ramu Ramanathan, Justin Rogers, Jacob Rossi, Reed Townsend and Steve Wright.</p>
     </section>
+    <section class="appendix">
+        <h1>Security and Privacy Considerations</h1>
+        <p>There are no known security or privacy impacts resulting from the features outlined in this specification.</p>
```

Comment:
I don't think a list of bullet points is required, but certainly there's some amount of fingerprinting possible by exposing touch events.